mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/Base64Test.java

```diff
@@ -0,0 +1,44 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.Base64Page;
+import io.restassured.response.Response;
+import org.junit.Test;
+
+import java.text.MessageFormat;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class Base64Test extends BasePageWebApiTest {
+	private Base64Page base64Page = new Base64Page();
+
+	@Test
+	public void base64DecodeAwesome() {
```

Comment:
Test name is still a method name, thus should be a verb. Please consider changing the name of both tests.